### ISSUE TITLE
Reevaluate Issue Templates and Labels

### ISSUE THREAD

**Issue by michaeljscrockett** [drnikki/text-ile#90]
Make more descriptive/helpful issue templates and labels, if needed. Look into creating issue forms.

**Comment by drnikki** [comment]
Yes agree!  @michaeljscrockett do you have thoughts on how detailed a scheme feels right to you? 

https://opensource.creativecommons.org/contributing-code/repo-labels/ for example is quite robust.
